orchagent/pfcwdorch.cpp

```diff
@@ -22,6 +22,7 @@
 #define PFC_WD_POLL_TIMEOUT             5000
 #define SAI_PORT_STAT_PFC_PREFIX        "SAI_PORT_STAT_PFC_"
 #define PFC_WD_TC_MAX 8
+#define COUNTER_CHECK_POLL_TIMEOUT_SEC  5
```

Comment:
prefer 1 second

Reply:
Is the polling interval controlled via cli or via config_db.json? If yes can you please explain how?

If not this needs fixing prior to committing.

Reply:
this is not controllable from the cli, this is only polling queue when the queue in pfc recovery state, in this case, we need to periodically poll it to get dropped packets. I do not see a generic need to make it controllable from cli.

Reply:
@lguohan All polling intervals need to be controlled via the cli and should provide the ability to be disabled.